HW_dev_pro/education_docker/web/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM   registry.access.redhat.com/ubi8/ubi:latest
+
+RUN yum install -y sudo && \
+    sudo dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
+    sudo dnf -y install https://rpms.remirepo.net/enterprise/remi-release-8.rpm && \
+    sudo dnf -y install php php-common php-process php-xmlrpc php-xml php-soap php-snmp php-recode php-bcmath php-cli php-dba php-dbg php-mbstring php-odbc php-pecl-apcu-devel php-pecl-zip php-pgsql php-pecl-apcu php-pear php-pdo php-opcache php-devel php-embedded php-enchant php-gd php-fpm php-gmp php-intl php-ldap php-json php-mysqlnd php-pdo php-gd php-mbstring zip unzip tar wget && \
+    sudo dnf -y install nginx nginx-all-modules && \
+    sudo curl -o phpMyAdmin-5.0.0-all-languages.zip https://files.phpmyadmin.net/phpMyAdmin/5.0.0/phpMyAdmin-5.0.0-all-languages.zip && \
```

Comment:
Your archive file stays inside image and is not being deleted
This add unnecessary size to the resulted image